wireapp/kalium
# feat: feature flags [AR-1463]

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

For the build configs that we generate from the client side we needed a way to send it to kalium and read it back , so we will have SSOT of the configs (build and run time )
for now we only have the build time , but later we will combine it with the run time 


### Solutions

created a data class for the configs that we pass when we initiate the coreLogic from the client side 
and create a use case that returns a flow of this data class 
### Dependencies (Optional)

https://github.com/wireapp/wire-android-reloaded/pull/636

Needs releases with:

https://github.com/wireapp/wire-android-reloaded/pull/636

### Testing


#### How to Test

after you initiate with some configs you can check if the use case returning these values 

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
